packages/e2e-tests/persistent-seeder-deployment/persistent-seeder.dockerfile

```diff
@@ -1,15 +1,8 @@
-FROM node:10.16.3
-
+FROM circleci/node:10.16.3-browsers
+USER root
```

Comment:
I remember seeing that this is generally bad practice. Probably not a big deal since this is all open source, but if we can avoid it perhaps we should?

Reply:
Yeah, I tried not using that, but the image wouldn't let me install xvfb for some reason.
Then I tried using a less privileged user for the rest of things, and ran into even more problems. On Monday I'll give it another look. I'm interested in that issue, I didn't like it either

Reply:
https://github.com/statechannels/monorepo/pull/1647/commits/6a08eccbff4bc9252c99657d75650cad004aad20